test/SlowTests/Issues/RavenDB-20731.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FastTests.Graph;
+using FastTests.Utils;
+using Microsoft.AspNetCore.Http.HttpResults;
+using NetTopologySuite.Utilities;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Server;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using Assert = Xunit.Assert;
+
+namespace SlowTests.Issues
+{
+    internal class RavenDB_20731 : ClusterTestBase
+    {
+        public RavenDB_20731(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ForceCreatedRevisionsShouldReplicate()
+        {
+            var (nodes, leader) = await CreateRaftCluster(numberOfNodes: 2, watcherCluster: true);
+            var databaseName = GetDatabaseName();
+            await CreateDatabaseInCluster(databaseName, 2, leader.WebUrl);
```

Comment:
use `GetDocumentStore` instead

Reply:
But I want stores for specific servers, it's more